cli/cmd/plugin/diagnostics/README.md

```diff
@@ -71,13 +71,14 @@ Flags:
       --management-cluster-name string         The name of the management cluster (required)
       --management-cluster-skip                If true, skips management cluster diagnostics
       --output-dir string                      Output directory for collected bundle (default "./")
-      --standalone-cluster-context string      The context name of the standalone cluster
-      --standalone-cluster-kubeconfig string   The standalone cluster config file (required) (default "${HOME}/.kube/config")
-      --standalone-cluster-name string         The name for the standalone cluster (required)
-      --work-dir string                        Working directory for collected data (default "${HOME}/.config/tanzu/diagnostics")
+      --unmanaged-cluster-context string       The context name of the unmanaged cluster
+      --unmanaged-cluster-kubeconfig string    The unmanaged cluster config file (required) (default "/home/jmcb/.kube/config")
```

Comment:
Sorry, one more - can we keep the `${HOME}` placeholder for the paths?

Reply:
🙈